pi3d/Display.py

```diff
@@ -383,12 +383,14 @@ def _loop_end(self):
     for sprite in to_unload:
       sprite.unload_opengl()
 
-    if getattr(self, 'frames_per_second', 0):
-      self.time += 1.0 / self.frames_per_second
-      delta = self.time - time.time()
+    if self.frames_per_second:
+      now = time.time()
+      delta = 1. / self.frames_per_second - (now - self.time)
```

Comment:
Two tiny quibbles!  :-)

1. You don't need the `.`
2. I don't really think that the code is made clearer by introducing the new local variable `now`.

(Ah, don't get me wrong, I love introducing local variables as a way to document... I simply believe that `time.time()` is so simple it doesn't need it.)

Reply:
1. You mean in `1.` ? For python2 compatibility you need it since there's no guarantee that `self.frames_per_seconds` is a float
2. fair enough !